documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -11,14 +11,18 @@ description = "Auxiliary images are an alternative approach for supplying a doma
  - [Introduction](#introduction)
  - [References](#references)
  - [Configuration](#configuration)
-   - [Auxiliary images](#auxiliary-images)
    - [Auxiliary volumes and paths](#auxiliary-volumes-and-paths)
+   - [Auxiliary images](#auxiliary-images)
    - [Model in Image paths](#model-in-image-paths)
+ - [Merge order](#merge-order)
+   - [Expected merge order](#expected-merge-order)
+   - [Performing replaces instead of merges](#performing-replaces-instead-of-merges)
+   - [Merge ordering example](#merge-ordering-example)
  - [Sample](#sample)
-    - [Step 1: Prerequisites](#step-1-prerequisites)
-    - [Step 2: Create the auxiliary image](#step-2-create-the-auxiliary-image)
-    - [Step 3: Prepare and apply the domain resource](#step-3-prepare-and-apply-the-domain-resource)
-    - [Step 4: Invoke the web application](#step-4-invoke-the-web-application)
+   - [Step 1: Prerequisites](#step-1-prerequisites)
+   - [Step 2: Create the auxiliary image](#step-2-create-the-auxiliary-image)
+   - [Step 3: Prepare and apply the domain resource](#step-3-prepare-and-apply-the-domain-resource)
+   - [Step 4: Invoke the web application](#step-4-invoke-the-web-application)
 
 ### Introduction
 
```

Comment:
The advantages of auxiliary image -> (Either) The advantages of an auxiliary image OR The advantages of auxiliary images